### PR TITLE
修改SegmentedControl默认颜色为主题颜色

### PR DESCRIPTION
**Pull request by jihfgz**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

File: components/segmented-control/segmented.android.tsx [hunk 1]
```diff
@@ -83,12 +82,13 @@ export default class SegmentedControl extends React.Component<
...
-    const { style, disabled, values = [], tintColor } = this.props;
-
+    const { style, disabled, values = [] } = this.props;
+    let { tintColor } = this.props;
     return (
       <WithTheme styles={this.props.styles} themeStyles={AndroidStyles}>
-        {styles => {
+        {(styles, theme) => {
           const selectedIndex = this.state.selectedIndex;
+          tintColor = tintColor || theme.brand_primary;
```
Comment (bang88) [inline components/segmented-control/segmented.android.tsx]: 这里建议加多一个变量到 主题里面去。
`segmented_tint_color: '#108ee9'`

Comment (bang88) [inline components/segmented-control/segmented.android.tsx]:
```suggestion
          tintColor = tintColor || theme.segmented_tint_color;
```

Comment (bang88) [reply components/segmented-control/segmented.android.tsx]: 算了。

**Comment by bang88** [comment]
close #173

File: components/segmented-control/segmented.android.tsx (hunk 1 above)
Comment (jihfgz) [reply components/segmented-control/segmented.android.tsx]: 看了下默认样式，里面有个变量
 segmented_control_color: brandPrimary,  // 同时应用于背景、文字颜色、边框色

Comment (bang88) [reply components/segmented-control/segmented.android.tsx]: 那这里可以改成用这个变量

**Pull request merged**